DB/api/db.py

```diff
@@ -43,18 +43,27 @@ def query(parsed_url):
 		device = parsed_url['device']
 	if 'diff' in parsed_url.keys():
 		diff = parsed_url['diff']
-
-	results = retrieve_within_filters(
-		device_id,
-		start_time,
-		end_time,
-		data_type,
-		subset,
-		limit,
-		reverse,
-		device,
-		diff,
-	)
+	if parsed_url['granularity']:
```

Comment:
Use the existing style, or if you think this style is better explain why and update the others to match.